src/main/java/Cloud.java

```diff
@@ -0,0 +1,5 @@
+public enum Cloud {
```

Comment:
Overall, your code follows the coding standard really well, making it easy to read and understand! :-) Classes are well implemented with clear distinction in each of their functions! A suggestion I have is to put a description of the function of this enum as it was not immediately clear of it's function! Otherwise, good implementation overall!